### PR TITLE
script to cleanup testing/metadata dir 

### PR DESCRIPTION
**Pull request by rohanreddych**
Issue #1534 --dry-run not included, will include in future PR. Did not get response in comments so decided to create a PR, this script needs some work, please be patient, check for the issues with this script in the comments of #1534 @Conchylicultor  any response would be appreciated.

### CODE REVIEW

**Comment by Conchylicultor** [comment]
Thank you for starting this. I answered on the bug. Please comment here to reassign to me when the script is updated.

**Comment by rohanreddych** [comment]
@Conchylicultor  still not getting any output from `iter_dataset_full_names`.
I think you have to remove the `registered` in 'registered._DATASET_REGISTRY` in the util you provided in the comment. You will get a `NameError`.
Even after removing this and adding `import tensorflow_datasets as tfds` i am getting an empty generator object.(I added it in `tensorflow_datasets/__init__.py` and `tensorflow_datasets/core/__init__.py` ).

File: tensorflow_datasets/__init__.py [hunk 1]
```diff
@@ -56,7 +56,7 @@
 from tensorflow_datasets import text
 from tensorflow_datasets import translate
 from tensorflow_datasets import video
-
+import tensorflow_datasets as tfds
```
Comment (Conchylicultor) [inline tensorflow_datasets/__init__.py]: What is this line supposed to achieve ? The module cannot import himself.

`import tensorflow_datasets as tfds` should be added to the `cleanup_script.py` script.

Comment (rohanreddych) [reply tensorflow_datasets/__init__.py]: >I added it in tensorflow_datasets/__init__.py
I am still getting an empty genarator object can you once run and check `iter_dataset_full_names` function. Thanks.

Comment (Conchylicultor) [reply tensorflow_datasets/__init__.py]: I do not see your modifications. Did you correctly imported the `import tensorflow_datasets as tfds` in `cleanup_script.py` ? When you're executing the `cleanup_script.py`, what is the output of `tfds.list_builders()` ?

**Comment by rohanreddych** [comment]
Ok closing this now

**Pull request closed without merging**